hummingbot/client/command/start_command.py

```diff
@@ -105,6 +130,8 @@ async def start_check(self,  # type: HummingbotApplication
                 self.notify(f"\nConnector status: {status}. This connector has one or more issues.\n"
                             "Refer to our Github page for more info: https://github.com/coinalpha/hummingbot")
 
+        self._initialize_notifiers()
```

Comment:
I believe `self._initialize_notifiers()` should be called at the previous location, right after disabling App Nap for MacOS clients.